packages/eslint-plugin-gestalt/src/button-icon-restrictions.test.js

```diff
@@ -20,7 +20,17 @@ const invalidWrongIcon = readFileSync(
   'utf-8',
 );
 
-const errorMessage = 'Buttons using iconEnd must use "arrow-down"';
+const invalidWrongIconLinkRole = readFileSync(
+  path.resolve(
+    __dirname,
+    './__fixtures__/button-icon-restrictions/invalid/invalid-wrong-link-role-icon.js',
+  ),
+  'utf-8',
+);
+
+const errorMessage = 'Buttons using iconEnd must use "arrow-down" icon';
```

Comment:
nit: you can save yourself some typing by exporting the error messages from the rule file

Reply:
DONE